src/main/webapp/index.html

```diff
@@ -4,21 +4,24 @@
     <meta charset="UTF-8">
     <title>Air Quality Map</title>
     <link rel="stylesheet" href="style.css">
```

Comment:
Prefer absolute references for included js/css files, as this ensures they stay correct if the document moves within the tree. ie prefer href="/style.css", with the leading "/".

Without a leading "/" these are relative references, eg href="style.css" will look for the file in the same path as the referring document.